src/executables/util_cs.c

```diff
@@ -4902,7 +4902,7 @@ flashback (UTIL_FUNCTION_ARG * arg)
 
   int trid = 0;
   int num_item = 0;
-  char *loginfo_list;
+  char *loginfo_list = NULL;
```

Comment:
Where can I find the `loginfo_list` is used? Because I try to compare the `loginfo_list ` value before it is not defined `NULL`

Reply:
Thanks, I will fix it. it was omitted when resolving a conflict error.